### ISSUE TITLE
Load and Save Profiles Error

### ISSUE THREAD

**Issue by jijames** [abrignoni/ALEAPP#281]
aleappGUI -> open the load/save profile dialog and click the X or cancel causes the app to crash in Linux.

```
$ python3 aleappGUI.py 
Traceback (most recent call last):
  File "/home/joshua/Documents/projects/ALEAPP/aleappGUI.py", line 160, in <module>
    if destination_path is not None and os.path.exists(destination_path):
  File "/usr/lib/python3.10/genericpath.py", line 19, in exists
    os.stat(path)
TypeError: stat: path should be string, bytes, os.PathLike or integer, not tuple
```

Tried setting destination_path manually, printing the type, didn't work.
The error seems to be with ```sg.popup_get_file``` returning something odd.

**Comment by cclgroupltd** [comment]
Weirdly this was only a problem for the save operation on Windows...

Either way, this is fixed as of 0fd97ab2a159f05c9854b52ba2af1757e8214633 I think? Can we close?

**Comment by jijames** [comment]
The fix worked on all my tests, and the update was ported to i/rleapp.
